cmd/commands/runtime_install.go

```diff
@@ -1009,12 +1012,12 @@ func installComponents(ctx context.Context, opts *RuntimeInstallOptions, rt *run
 	var err error
 
 	if !store.Get().SkipIngress && rt.Spec.IngressController != string(ingressutil.IngressControllerALB) {
-		if err = createWorkflowsIngress(ctx, opts, rt); err != nil {
+		if err = tryToCreateWorkflowsIngress(ctx, opts, rt); err != nil {
```

Comment:
the function names can remain the same. functions always "try" to do something, and they might fail and return an error.
the only change is that now, inside the function, there is a retry loop.